README.md

```diff
@@ -1,8 +1,41 @@
+# Transit Movements Auditing
+This is a microservice for an internal API to audit events.
```

Comment:
This microservice provides the capability for private services to audit internal events.